syntax = "proto3";
package redis;

// Service
service redisCache {
  rpc Read(ReadRequest) returns (ReadResponse) {}
  rpc Write(WriteRequest) returns (WriteResponse) {}

}

// Request Name
message ReadRequest {
  string query=1;
  bool force=2;
}

message WriteRequest {
  bytes query=1;
}


// Response Array
message ReadResponse{
  bytes data=1;
}

message WriteResponse{
  string key=1;
}